app/javascript/components/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+// import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Greeting from './Greeting';
+
+const App = () => (
+  <div>
+    <Greeting />
+  </div>
+);
+
+export default App;
```

Comment:
- [ ] Instead of just importing the Greeting component and rendering it here, you are required to create a route for it.

- [ ] Kindly avoid having commented code as it can make your code hard to read.